File: Modul-4/C/PRAK401-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int x, i;
    char simbol[10];
    printf("Masukkan Bilangan Kelipatan dan Simbol Pengganti : \n");
    scanf( "%d %s", &x, &simbol);

    printf("Bilangan Kelipatan yang Sudah Dirubah Menjadi Simbol : \n");

    for( i = 1; i < 51; i++ )
    {
        if(i % x == 0)
        {
            printf("%s ", simbol);
        }
        else
        { 
            printf("%d ", i);
        }
    }
    return 0;
}

File: Modul-4/Python/PRAK401-2310817320007-ZahraNabila.py
x, simbol = input("Masukkan Bilangan Kelipatan dan Simbol Pengganti : \n").split()
x = int(x)

print("Bilangan Kelipatan yang Sudah Dirubah Menjadi Simbol   :")

for i in range(1, 51):
    if i % x == 0:
        print(simbol, end=" ")
    else:
        print(i, end=" ")